backend/tracim_backend/views/core_api/user_controller.py

```diff
@@ -320,6 +330,8 @@ def set_user_email(self, context, request: TracimRequest, hapic_data=None):
         uapi = UserApi(
             current_user=request.current_user, session=request.dbsession, config=app_config  # User
         )
+        if not has_write_rights("email", app_config, request):
+            return HTTPBadRequest()
```

Comment:
You must return some explanation. Error cases must be conform to Tracim error code strategy: You must return a Tracim error code + details. See `error.py` + `class ErrorCode`